nix/shells.nix

```diff
@@ -75,8 +75,6 @@ rec {
     ]);
   });
 
-
```

Comment:
this whole file can be deprecated. we're using https://github.com/holochain/holochain/blob/develop/nix/modules/devShells.nix now